Makefile

```diff
@@ -45,93 +41,29 @@ show-lint:
 	rm -f /tmp/autograph-golint.txt /tmp/autograph-crypto11-check.txt
 
 vet:
-	go vet $(PACKAGE_PATHS)
+	go vet $(PACKAGE_NAMES)
 
 fmt-diff:
 	gofmt -d *.go database/ signer/ tools/autograph-client/ $(shell ls tools/autograph-monitor/*.go) tools/softhsm/ tools/hawk-token-maker/ tools/make-hsm-ee/ tools/makecsr/ tools/genpki/
 
 fmt-fix:
-	go fmt $(PACKAGE_PATHS)
+	go fmt $(PACKAGE_NAMES)
 	gofmt -w tools/autograph-client/ $(shell ls tools/autograph-monitor/*.go) tools/softhsm/ tools/hawk-token-maker/ tools/make-hsm-ee/ tools/makecsr/ tools/genpki/
 
-# for test* and showcoverage* targets include known package names for
-# autocompletion but fall through to generic implementation
-
-# non-signer package paths
-testautograph: PACKAGE_PATH = go.mozilla.org/autograph
-testdatabase: PACKAGE_PATH = go.mozilla.org/autograph/database
-testformats:  PACKAGE_PATH = go.mozilla.org/autograph/formats
-testmonitor: PACKAGE_PATH = go.mozilla.org/autograph/tools/autograph-monitor
-testsigner: PACKAGE_PATH = go.mozilla.org/autograph/signer
-
-testapk:
-testcontentsignature:
-testcontentsignaturepki:
-testgenericrsa:
-testgpg2:
-testmar:
-testpgp:
-testrsapss:
-testxpi:
-# default vars for test* targets https://www.gnu.org/software/make/manual/html_node/Pattern_002dspecific.html#Pattern_002dspecific
-test%: PACKAGE_NAME = $(subst test,,$@)
-test%: PACKAGE_PATH = $(addprefix go.mozilla.org/autograph/signer/,$(subst test,,$@))
-test%: PACKAGE_TEST_OUTPUT_DIR = $(subst test,testprofiles/,$@)
-test%:
-	mkdir -p $(PACKAGE_TEST_OUTPUT_DIR)
-	$(GOTEST) -outputdir "testprofiles/$(PACKAGE_NAME)" $(PACKAGE_PATH)
-ifeq ($(RACE_TEST),1)
-	$(GO) test -v -race $(PACKAGE_PATH)
-endif
-ifeq ($(STATIC_CHECK),1)
-	$(GO) staticcheck $(PACKAGE_PATH)
```

Comment:
note to @g-k: make issues for adding race and staticchecks back

Reply:
oh that's a whoops. I'll add them back!